htdocs/core/js/lib_foot.js.php

```diff
@@ -245,3 +245,59 @@ function() {
 		lastchild.innerHTML = \''.dol_escape_js($langs->trans('CopiedToClipboard')).'\';
 		setTimeout(() => { lastchild.innerHTML = tmp; }, 1000);
 	})'."\n";
+
+
+print "\n/* JS CODE TO ENABLE ClipBoard copy paste*/\n";
```

Comment:
Is comment still valid ? May be

`print "\n/* JS CODE TO ENABLE DIALOG CONFIRM POPUP ON ACTION BUTTON */\n";`

Reply:
Oups, thank !
Corrected